ui/app/components/app/account-menu/account-menu.component.js

```diff
@@ -126,6 +132,23 @@ export default class AccountMenu extends PureComponent {
               type={PRIMARY}
             />
           </div>
+          { iconAndNameForOpenDomain
+            ? <div className="account-menu__icon-list">
+              <div className="connected-sites-list__identicon-container">
```

Comment:
Everything inside the `connected-sites-list__identicon-container` div should be made into a component.

Reply:
bbc8d9a9c